tox-pip9.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Can we avoid adhoc sh scripts since we have Makefile?

Reply:
I guess we can use make for that. But I consider Makefile to be a list of commands useful for a developer, and tox-pip9 is definitely not one.

Are you sure that it should be moved to the Makefile?

Reply:
I'm not sure about Makefile, but I'm sure that there is no place for custom scripts in project root. Can we have this defined in tox.ini itself? All toxic things in a single place sounds as good idea.

Also, remind me, why should we care about pip version?

Reply:
> Can we have this defined in tox.ini itself?

I don't think that's possible. Commands in tox file are not interpreted with shell, so the only option I see to run pip install twice is to introduce another command/script.

> All toxic things in a single place sounds as good idea.

I don't think this is an appropriate wording for that kind of things, although I admire the pun. 

I would prefer to keep this script in the root; having it in Makefile also sounds like a valid option to me. I don't see any other options which might seem as nearly appropriate as these two: creating a separate directory just for this script definitely doesn't look like one.

> Also, remind me, why should we care about pip version?

It's written in the comments: https://github.com/rambler-digital-solutions/airflow-declarative/blob/38c838ee1f49051dde2f83e0313cf4c74a7539cc/tox.ini#L3-L4

Reply:
I figured that the obvious `bash -c '...'` options is not discussed here. The problem with it is that tox, apparently, cannot pass their substitution marks `{opts}` and `{packages}` while keeping them in the same positional arg (i.e. the value for the `-c` key). It's assumed in the code [1] that these would be their own positional args and I didn't manage to get it working as well. 

Perhaps there's an option to pass positional args which I'm not aware of? I'm thinking about something like `bash -c 'pip install $1' {packages}`

\[1\]: https://github.com/tox-dev/tox/blob/9b1802d4a62a60339ba58c2f22b1f4f9b9ac987f/src/tox/venv.py#L357

Reply:
Hm...ok. It seems like to be not a trivial change. Or at least it requires some investigations. Let's move on and fix that later. I think with drop 1.8 support one day things will get simplifies by themself.

Reply:
> I think with drop 1.8 support one day things will get simplifies by themself.

Indeed, that would be the easiest solution.